test/unit_tests/braket/experimental/autoqasm/test_gate_calibrations.py

```diff
@@ -0,0 +1,255 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+"""Tests for the pulse control module."""
+
+import textwrap
+
+import pytest
+
+import braket.experimental.autoqasm as aq
+from braket.experimental.autoqasm import errors, pulse
+from braket.experimental.autoqasm.instructions import h, rx
+
+
+def test_gate_calibrations_fixed_args():
+    """test gate calibrations with fixed args"""
+
+    @aq.pulse_sequence(implements=h, target="$0")
+    def cal_1():
+        pulse.barrier(0)
```

Comment:
I see that qubit index `0` here creates a barrier on physical qubit `$0`. UX-wise this is a little confusing, as in other contexts `0` would be a virtual qubit. Would there be a way to make these consistent (or does this require a different way of representing physical qubits)?

Reply:
It was because of some implementation detailed. Fixed.